src/components/sidebar/routePathView/RoutePathViewForm.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import InputContainer from '../InputContainer';
+import { Button, Dropdown, Checkbox } from '../../controls';
+import ButtonType from '../../../enums/buttonType';
+import * as s from './routePathView.scss';
+
+interface IRoutePathViewFormState {
+}
+
+interface IRoutePathViewFormProps {
+    isEditingDisabled: boolean;
+}
+
+class RoutePathViewForm extends React.Component<IRoutePathViewFormProps, IRoutePathViewFormState>{
+    constructor(props: any) {
+        super(props);
+
+    }
+
+    // TODO
+    public onChange = () => {
+    }
+
+    public render(): any {
+        const isEditingDisabled = this.props.isEditingDisabled;
+
+        return (
+        <div>
+            <div className={s.routePathTimestamp}>01.09.2017</div>
+            <div className={s.padding} />
+            <div className={s.padding} />
+            <div className={s.topic}>
+                REITIN OTSIKKOTIEDOT
```

Comment:
since we use this a lot why not have `<FormTopic text="reitin otsikkotiedot">` and make the text capitalized in css.

Reply:
Okay realised this is in another issue (#283) already